FoodBookApp/UI/Views/CreateReview/CreateReview2View.swift

```diff
@@ -105,6 +105,18 @@ struct CreateReview2View: View {
                         }
                     }.padding().background(customGray2).cornerRadius(10)
                 }.padding(.horizontal).padding(.top)
+                .navigationTitle("Review")
+                .toolbar{
+                    Button(action: {
```

Comment:
This button has no navigation (see video). Not sure if it's on purpose bc we're waiting for the logic behind.

https://github.com/ISIS3510-202410-Team23/SwiftApp/assets/69609680/5cc9dc9e-acd7-466d-b7b0-afcf0ff88644

Reply:
This I agree should be resolved.

Reply:
Will try to resolve with just closing the sheet, once we implemente business logic it should obvs do the actual review creation.